tests/test_bears/AspectTestBear.py

```diff
@@ -0,0 +1,27 @@
+from coalib.bearlib.aspects import Root
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+class AspectTestBear(LocalBear, aspects={
+    'detect': [
+        Root.Redundancy.UnusedVariable.UnusedGlobalVariable,
+        Root.Redundancy.UnusedVariable.UnusedLocalVariable,
+    ]
+}, languages=['Python']):
```

Comment:
IMO that was weird too :/

I think this update is a bit better. Also I would like to avoid using `get_aspect` for better clarity at first glance about bear's aspect (the first time is because I was too lazy to search the correct fullname :sweat_smile: ).